### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by nagi-desuuu**
- Adds skinning info for newly-added elements.
- Modify existing documentation for grammar errors and miscellaneous corrections.
- Split off skinning sections into their own pages.
- More changes that are too long for this summary, so it's advised to read through the changes.

### CODE REVIEW

File: skinning/json.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    "#f7314f"
+  ]
+}
+```
+
+- `column_width` decides the width of all the columns in pixels.
+- `hit_position` decides how many pixels up from the bottom of the screen the hit position is.
+- `colors` tints the hit lighting of each individual column. The first entry will tint the lighting in the first lane from the left of the stage, the second entry will tint the second lane, and so on.
+
+This goes from 1K to 10K so all keymodes will need to be added.
```
Comment (flustix) [inline skinning/json.md]: 1-3k and 9-10k only are accessible thru osu!mania maps. And it doesn't really *need* to be added.

File: skinning/json.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+It's advised to keep the values the same between keymodes to avoid awkward looking transitions. These values will also scale all of the above images, so try to get the image width to match the column width in the .json file to make editing and setting the hit position easier.
+
+### Judgement Colours
+Also needed in the .json file are the judgement colours. These decide what colour the Judgement Splash is when displayed.
+
+The default colours for each judgement are as follows:
+
+|Judgement|Old Defaults|New Defaults|
```
Comment (flustix) [inline skinning/json.md]: "Old defaults"? They always have been the "New Defaults". The "old" ones are from the example skin and never actually have been in the game.

File: skinning/judgement.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Judgement
+These are the files that determine what image shows up for each of the judgements in-game. The files are labled as such:
+
+|Filename|Judgement|
+|---|---|
+|`flawless.png`|Maximum Hit Judgement|
+|`perfect.png`|300 Hit Judgement|
```
Comment (flustix) [inline skinning/judgement.md]: These numbers are from osu!. They have nothing to do with this.

File: skinning/judgement.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Judgement
+These are the files that determine what image shows up for each of the judgements in-game. The files are labled as such:
```
Comment (flustix) [inline skinning/judgement.md:2]: Maybe mention that they should be in a folder. (Applies to all other sections too)

File: skinning/lighting.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Lighting
+This is what decides what image shows up when hitting each column. This is labelled as `column-lighting.png`, and is applied to all columns regardless of gamemode.
+
+It is possible to tint each column's lighting individually by following the skin.json commands.
```
Comment (flustix) [inline skinning/lighting.md]: Might be unclear. Maybe mention the "Colors" section.

File: skinning/samples.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Samples
+
+You can customize the sounds for when a key is pressed, you miss a note, etc.
+
+|Sample|Usage|
+|---|---|
+|`fail.wav`|Failing for any reason.|
```
Comment (flustix) [inline skinning/samples.md:7]: Gameplay samples are in a subfolder. ("Samples/Gameplay")

File: skinning/stage.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Stage
+These decide how the non-playfield elements are rendered in game. These include `border-left.png` and `border-right.png` which decide how the left and right side of the playfield look like, and `hitline.png` which shows where the hitwindow is for notes.
```
Comment (flustix) [inline skinning/stage.md:2]: background.png is missing here.

File: skinning/json.md (hunk 2 above)
Comment (nagi-desuuu) [reply skinning/json.md]: Should I use them in the template then?

File: skinning/judgement.md (hunk 3 above)
Comment (nagi-desuuu) [reply skinning/judgement.md]: Should I just copy this game's judgement names?

File: skinning/json.md (hunk 2 above)
Comment (flustix) [reply skinning/json.md]: Maybe just remove the "Old" Column and rename "New Defaults" to "Color"

File: skinning/judgement.md (hunk 3 above)
Comment (flustix) [reply skinning/judgement.md]: Should be fine yeah

**Pull request merged**